src/api/procedures/__tests__/leaveIdentity.ts

```diff
@@ -70,10 +70,10 @@ describe('modifyCaCheckpoint procedure', () => {
       error = err;
     }
 
-    expect(error.message).toBe('Only Secondary Keys are allowed to leave an Identity');
+    expect(error.message).toBe('Only Secondary Accounts are allowed to leave an Identity');
```

Comment:
nit-> Here **S**econdary is in Title case but in all other places (JS Docs and other references it wasn't). We should be uniform

Reply:
Good point,I agree with being consistent. I went with "secondary Account".

I like the look of "Secondary Account" better, but think we should stick with the rule of entities getting the capital letter, and there isn't a "SecondaryAccount" entity.